openshift/oc-mirror
# fix(create): add metadata rollback for errors after metadata write in create operation

# Description

This PR will add functionality that will roll back the metadata in the event an error occurs during archiving or the user cancels during archiving. It also moves `testdata` for each package under the package root.

Fixes #200 

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

- [X] Unit test for Create with Cancel

Instructions to manually test:
1. Use stateless mode
2. Run a an oc-mirror mirror to disk operation
3. Cancel the operation during archiving
4. Ensure that when you run again there are not sequence errors

# Checklist:
